lightspeed/modules/con_training-data.adoc

```diff
@@ -6,9 +6,17 @@
 
 == Models
 
-{LightspeedFullName} uses Ansible-specific IBM watsonx Granite models unique to your organization. These models are provided, managed, and maintained by IBM. 
+{LightspeedFullName} uses Ansible-specific IBM watsonx Granite models unique to your organization. These models are provided, managed, and maintained by IBM. You can use the Ansible-specific IBM watsonx Granite model as the default model for{LightspeedShortName}, so that all Ansible Lightspeed users in your organization can use the fine-tuned model. 
```

Comment:
Existing: "default model for{LightspeedShortName}"
Proposed: "default model for {LightspeedShortName}" 
(spacing)

Reply:
Updated.